src/openforms/js/public.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('DOMContentLoaded', event => {
```

Comment:
so turns out that this approach is to be preferred:

```
if (document.readyState !== 'loading') {
  init();
} else {
  document.addEventListener('DOMContentLoaded', init);
}
```

... probably we should write a utility `onLoaded` function for that so we can just do `onLoaded(init);` at the end of the module.